lib/models/profile.dart

```diff
@@ -0,0 +1,36 @@
+enum Gender {
+  male,
+  female,
+}
+
+class Profile {
```

Comment:
It is a good practice to add json serialization methods to your model class; such as: `toJson` and `fromJson`.

The class that should know how to convert between json and `Profile` object is this `Profile` class. 
Currently the conversion is done inside `ChangeNotifier` class which should only care about app business logic

Reply:
added the json serialization methods